nilearn/plotting/surf_plotting.py

```diff
@@ -87,12 +94,15 @@ def niimg_to_surf_data(image, mesh_nodes, ball_radius=3.):
 
     Parameters
     ----------
-    image: Nifti image, 3d or 4d.
-    mesh_nodes: array-like,shape n_nodes * 3
+
+    image : Nifti image, 3d or 4d.
```

Comment:
this does not yet seem to make the standard mention of nii-like image compatability